styles/styles.css

```diff
@@ -97,7 +97,423 @@ div ul li {
+  position: relative;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url(/img/odinproject.jpg);
+  background-size: cover;
+  display: grid;
+  padding: 120px 24px;
+  gap: 12px;
+  align-content: center;
+}
+
+.protitle {
+  color: white;
+  width: 295px;
+  height: 35px;
+  margin-top: 100px;
+}
+
+.prodetails {
+  width: 295px;
+  height: 87px;
+  color: white;
+}
+
+.proitem1 {
+  border: none;
+  color: white;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.proitem2 {
+  border: none;
+  color: white;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.proitem3 {
+  border: none;
+  color: white;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.probutton {
+  margin-top: -50px;
+  width: 327px;
+  z-index: 1;
+}
+
+#project2 {
+  width: 327px;
+  height: 386px;
+  position: relative;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url(/img/cafemenu.jpg);
+  background-size: cover;
+  display: grid;
+  padding: 120px 24px;
+  gap: 12px;
+  align-content: center;
+}
+
+.pro2title {
+  margin-top: 0;
+}
+
+.pro2flex {
+  margin-top: 60px;
+}
+
+.projsecondpage {
+  display: grid;
+  width: 100vw;
+  height: 100vh;
+  background-color: #e5e5e5;
+  padding: 120px 24px;
+  gap: 12px;
+  align-content: center;
+}
+
+#project3 {
+  width: 327px;
+  height: 386px;
+  position: relative;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url(/img/catphoto.jpg);
+  background-size: cover;
+  display: grid;
+  padding: 120px 24px;
+  gap: 12px;
+  align-content: center;
+}
+
+#project4 {
+  width: 327px;
+  height: 386px;
+  position: relative;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url(/img/photogallery.jpg);
+  background-size: cover;
+  display: grid;
+  padding: 120px 24px;
+  gap: 12px;
+  align-content: center;
+}
+
+.projthirdpage {
+  display: grid;
+  width: 100vw;
+  height: 100vh;
+  background-color: #e5e5e5;
+  padding: 120px 24px;
+  gap: 12px;
+  align-content: center;
+}
+
+#project5 {
+  width: 327px;
+  height: 386px;
+  position: relative;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url(/img/quiz.jpg);
+  background-size: cover;
+  display: grid;
+  padding: 120px 24px;
+  gap: 12px;
+  align-content: center;
+}
+
+#project6 {
+  width: 327px;
+  height: 386px;
+  position: relative;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url(/img/tributepage.jpg);
+  background-size: cover;
+  display: grid;
+  padding: 120px 24px;
+  gap: 12px;
+  align-content: center;
+}
+
+#aboutme {
+  display: flex;
+  flex-direction: column;
+  width: 100vw;
+  height: 100vh;
+  background-color: #1c1a19;
+  align-content: center;
+  padding: 120px 24px;
+  border-bottom: 1px solid white;
+  top: calc(50% - 0px / 2 + 0.5px);
+}
```

Comment:
- [ ] I would implore you to add a background image to this about section just as it is on the Figma design.

### Your design
![image](https://user-images.githubusercontent.com/86200356/219903555-904293ef-d4b7-4c5c-9969-0cc4d82bdc72.png)

### Figma Design
![image](https://user-images.githubusercontent.com/86200356/219903574-cc72531c-cd07-4da4-86ca-6be73ec6711f.png)